trust-quorum/src/node.rs

```diff
@@ -76,15 +76,70 @@ impl Node {
         Ok(persistent_state)
     }
 
+    /// Process a timer tick
+    ///
+    /// Ticks are issued by the caller in order to move the protocol forward.
+    /// The current time is passed in to make the calls deterministic.
+    pub fn tick(&mut self, now: Instant, outbox: &mut Vec<Envelope>) {
+        self.send_coordinator_msgs(now, outbox);
+    }
+
+    /// Handle a message from a another node
+    pub fn handle(
+        &mut self,
+        _now: Instant,
+        _outbox: &mut Vec<Envelope>,
+        from: PlatformId,
+        msg: PeerMsg,
+    ) -> Option<PersistentState> {
+        match msg {
+            PeerMsg::PrepareAck(epoch) => {
+                self.handle_prepare_ack(from, epoch);
+                None
+            }
+            _ => todo!(),
```

Comment:
mb something like:

```suggestion
            _ => todo!("don't yet know how to handle {msg:?}"),
```

Reply:
Done in [50ebf64](https://github.com/oxidecomputer/omicron/pull/8000/commits/50ebf6479db5b3e8e345394a1bcf2943306edf78)